config/software/postgresql.rb

```diff
@@ -70,4 +72,26 @@
   delete "#{install_dir}/embedded/lib/libpgport.a"
   delete "#{install_dir}/embedded/lib/libpgtypes.a"
   delete "#{install_dir}/embedded/lib/libpq.a"
+
+  # Delete postgres' binaries, except for pg_config which is required by psycopg's build
+  delete "#{install_dir}/embedded/bin/postgres"
+  delete "#{install_dir}/embedded/bin/ecpg"
+  delete "#{install_dir}/embedded/bin/pg_dump"
+  delete "#{install_dir}/embedded/bin/pg_restore"
+  delete "#{install_dir}/embedded/bin/pgbench"
+  delete "#{install_dir}/embedded/bin/pg_upgrade"
+  delete "#{install_dir}/embedded/bin/pg_basebackup"
+  delete "#{install_dir}/embedded/bin/pg_dumpall"
+  delete "#{install_dir}/embedded/bin/pg_waldump"
+  delete "#{install_dir}/embedded/bin/pg_rewind"
+  delete "#{install_dir}/embedded/bin/pg_recvlogical"
+  delete "#{install_dir}/embedded/bin/pg_receivewal"
+  delete "#{install_dir}/embedded/bin/pg_ctl"
+  delete "#{install_dir}/embedded/bin/pg_resetwal"
+  delete "#{install_dir}/embedded/bin/pg_isready"
+  delete "#{install_dir}/embedded/bin/pg_controldata"
+  delete "#{install_dir}/embedded/bin/pgrep"
```

Comment:
This one, `pgrep` is not part of postgres, and it was already in the build. Not sure if we need to keep it.

Reply:
woops indeed, it definitely doesn't belong in this PR. Thanks for spotting! 

I'm not sure it belongs in the artifact neither, but that's another package we'll need to check

Reply:
Found some more that are part of postgres:

```
psql
initdb
reindexdb
vacuumdb
createuser
clusterdb
createdb
dropuser
dropdb
```